### ISSUE TITLE
Xiaomi Mi A2 (jasmine sprout): Camera app freezes

### ISSUE THREAD

**Issue by Fazarel** [ubports/ubuntu-touch#1901]
- Device: Xiaomi Mi A2 (jasmine_sprout)
- Channel: stable
- Build: OTA-20 (20211109-1120)

### Steps to reproduce
- Take some photos

### Expected behavior
- Photos are taken

### Actual behavior
- The camera application randomly freezes either after the first photo or after taking several photos
- Most of the time, phone needs to be restarted to use the camera app again
- Sometimes after a while the camera application is usable again

### Logfiles and additional information
https://dpaste.com/AJCWK3H6M
Could be related to #959

**Comment by jefrisingh** [comment]
Yes, I also faced the same, I took 4 photos continuesly , on 4th photo it freeze and the photo is not saved, and I need to close and reopen it.

**Comment by matzgewinn** [comment]
same with my device. It is getting worse since a month.

**Comment by Sladek90** [comment]
Mee too. I have the same experience.
